sdk/core/src/http_client.rs

```diff
@@ -59,13 +61,13 @@ pub trait HttpClient: Send + Sync + std::fmt::Debug {
     }
 }
 
-pub static EMPTY_BODY: [u8; 0] = [];
+pub static EMPTY_BODY: Bytes = Bytes::from_static(&[]);
```

Comment:
I'm gong to change this back and use `Bytes::from_static(EMPTY_BODY)` instead of `Bytes::from(EMPTY_BODY.clone())`. I think it just reads better.